tests/h/views/api/groups_test.py

```diff
@@ -184,6 +186,78 @@ def pyramid_request(self, pyramid_request, factories):
         return pyramid_request
 
 
+@pytest.mark.usefixtures('group_service',
+                         'user_service',
+                         'request_auth_client',
+                         'validate_auth_client_authority')
+class TestAddMember(object):
+
+    def test_it_adds_current_user_to_group(self,
```

Comment:
This test name is not quite accurate—it's not adding the current user to the group, it's using the user indicated by the request params to the group